app/Http/Controllers/Auth/AuthController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Auth\LoginRequest;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Http\Request;
+
+class AuthController extends Controller
+{
+    public function login(LoginRequest $request)
+    {
+        $credentials = $request->only('email', 'password');
+        if (Auth::attempt($credentials)) 
+        {
+            return response()->json(['errors' => false]);
+        }
```

Comment:
We only place the `{` (open parenthesis) below if it's a class/function, not for conditional statements/loops

```suggestion
        if (Auth::attempt($credentials)) {
            return response()->json(['errors' => false]);
        }
```

Reply:
Noted on this ma'am. Thank your for reminding.